webapp/pages/profile/_id/_slug.vue

```diff
@@ -89,65 +89,19 @@
         <ds-heading tag="h3" soft style="text-align: center; margin-bottom: 10px;">
           {{ $t('profile.network.title') }}
         </ds-heading>
-        <base-card style="position: relative; height: auto;">
-          <ds-space v-if="user.following && user.following.length" margin="x-small">
-            <ds-text tag="h5" color="soft">
-              {{ userName | truncate(15) }} {{ $t('profile.network.following') }}
-            </ds-text>
-          </ds-space>
-          <template v-if="user.following && user.following.length">
-            <ds-space v-for="follow in uniq(user.following)" :key="follow.id" margin="x-small">
-              <!-- TODO: find better solution for rendering errors -->
-              <client-only>
-                <user-teaser :user="follow" />
-              </client-only>
-            </ds-space>
-            <ds-space v-if="user.followingCount - user.following.length" margin="small">
-              <ds-text size="small" color="softer">
-                {{
-                  $t('profile.network.andMore', {
-                    number: user.followingCount - user.following.length,
-                  })
-                }}
-              </ds-text>
-            </ds-space>
-          </template>
-          <template v-else>
-            <p style="text-align: center; opacity: 0.5;">
-              {{ userName }} {{ $t('profile.network.followingNobody') }}
-            </p>
-          </template>
-        </base-card>
+        <follow-list
+          :loading="$apollo.loading"
+          :user="user"
+          type="followedBy"
+          @fetchAllConnections="fetchAllConnections"
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-03-31T12:02:45Z" title="Tuesday, March 31st 2020, 2:02:45 pm +02:00">Mar 31, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -89,65 +89,19 @@
         <ds-heading tag="h3" soft style="text-align: center; margin-bottom: 10px;">
           {{ $t('profile.network.title') }}
         </ds-heading>
-        <base-card style="position: relative; height: auto;">
-          <ds-space v-if="user.following && user.following.length" margin="x-small">
-            <ds-text tag="h5" color="soft">
-              {{ userName | truncate(15) }} {{ $t('profile.network.following') }}
-            </ds-text>
-          </ds-space>
-          <template v-if="user.following && user.following.length">
-            <ds-space v-for="follow in uniq(user.following)" :key="follow.id" margin="x-small">
-              <!-- TODO: find better solution for rendering errors -->
-              <client-only>
-                <user-teaser :user="follow" />
-              </client-only>
-            </ds-space>
-            <ds-space v-if="user.followingCount - user.following.length" margin="small">
-              <ds-text size="small" color="softer">
-                {{
-                  $t('profile.network.andMore', {
-                    number: user.followingCount - user.following.length,
-                  })
-                }}
-              </ds-text>
-            </ds-space>
-          </template>
-          <template v-else>
-            <p style="text-align: center; opacity: 0.5;">
-              {{ userName }} {{ $t('profile.network.followingNobody') }}
-            </p>
-          </template>
-        </base-card>
+        <follow-list
+          :user="user"
+          type="followedBy"
+          @fetchAllConnections="fetchAllConnections"
```


same as :point_up_2:

Reply:
> [<img alt="rbeer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/rbeer) **Authored by [rbeer](https://github.com/rbeer)**
_<time datetime="2020-04-01T07:04:27Z" title="Wednesday, April 1st 2020, 9:04:27 am +02:00">Apr 1, 2020</time>_

---


done in 8e98570